mobile/src/main/java/ch/epfl/sdp/mobile/state/HomeActivity.kt

```diff
@@ -36,17 +37,18 @@ class HomeActivity : ComponentActivity() {
     val chessFacade = ChessFacade(auth, store, assetManager)
     val speechFacade = SpeechFacade(AndroidSpeechRecognizerFactory(this))
     val tournamentFacade = TournamentFacade(auth, dataStoreFactory, store, SystemTimeProvider)
+    val settingsFacade = SettingsFacade(dataStoreFactory)
 
     setContent {
       PawniesTheme {
-        ProvideLocalizedStrings {
+        ProvideLocalizedStrings(settingsFacade = settingsFacade) {
           ProvideFacades(
               authentication = authenticationFacade,
               social = socialFacade,
               chess = chessFacade,
               speech = speechFacade,
               tournament = tournamentFacade,
-          ) { Navigation() }
+              settings = settingsFacade) { Navigation() }
```

Comment:
If the facade is provided as a default argument, you call `ProvideLocalizedStrings` inside of `ProvideFacades` and remove the `settingsFacade` argument,

Reply:
done